flink-runtime-web/src/test/resources/rest_api_v1.snapshot

```diff
@@ -1968,7 +1968,7 @@
               "task" : {
                 "type" : "string"
               },
-              "location" : {
+              "endpoint" : {
```

Comment:
what happens if location and endpoint are specified?

Reply:
Nice catch, JsonAlias is used in this case and if users directly use the class from Flink v1.19 both can be parsed. However if users use a custom parsing logic or use the class from an older version of Flink, this will break. I'll change it to keep both location and endpoint fields present.